bin/epaa.py

```diff
@@ -1051,6 +1051,8 @@ def __main__():
             pred_dataframes, statistics = make_predictions_from_peptides(peptides, methods, thresholds, alleles, up_db, args.identifier, metadata)
         else:
             pred_dataframes, statistics, all_peptides_filtered, proteins = make_predictions_from_variants(vl, methods, thresholds, alleles, int(args.min_length), int(args.max_length) + 1, ma, up_db, args.identifier, metadata, transcriptProteinMap)
+
+    no_predictions = False
```

Comment:
Would it make more sense to define the boolean within the try clause for reading purposes?